python/semantic_kernel/connectors/ai/ollama/ollama_prompt_execution_settings.py

```diff
@@ -27,6 +29,12 @@ class OllamaTextPromptExecutionSettings(OllamaPromptExecutionSettings):
 class OllamaChatPromptExecutionSettings(OllamaPromptExecutionSettings):
     """Settings for Ollama chat prompt execution."""
 
+    tools: list[dict[str, Any]] | None = Field(
+        None,
+        max_length=64,
```

Comment:
Curious, is this the actual limit from Ollama or just the default we've been using?

Reply:
This is the default we have been using. @eavanvalkenburg, @moonbox3 may know the history?

Reply:
at the time this was the max for OpenAI function calling, might come from there.

Reply:
I think the setting should either be appropriate for the AI service connector or we don't include it so we don't need to keep up with the service as they changed (for example, OpenAI now supports 128 tools from 64).

Reply:
Will address this issue in another task.